docs/setup.en.md

```diff
@@ -62,6 +62,13 @@ npm install web-push -g
 web-push generate-vapid-keys
 ```
 
+*(optional)* Create a twitter application
+----------------------------------------------------------------
+If you want to enable the twitter integration, you need to create a twitter app at [apps.twitter.com](https://apps.twitter.com/).
```

Comment:
<https://apps.twitter.com> was deprecated. Please replace to <https://developer.twitter.com/en/apps>.